### PR TITLE
complete exercise

### PR DESCRIPTION
**Pull request by aalvar0**

### CODE REVIEW

File: exercise_02/getTotalWordsFromString.js [hunk 1]
```diff
@@ -3,3 +3,16 @@
  * input: string (sentence)
  * output: number
  */
+
+function getTotalWordsFromString(sentence) {
+    const trimmedSentence = sentence.trim();
```
Comment (tictools) [inline exercise_02/getTotalWordsFromString.js]: Do you know how does `trim` work? What does it return? 🤔

File: exercise_07/isTypeOf.js [hunk 2]
```diff
@@ -6,3 +6,23 @@
...
  * output:
  */
+
+// isTypeOf.js
+function isTypeOf(value, type) {
+  if (type === "array") {
+    return Array.isArray(value);
+  }
+
+  switch (type) {
```
Comment (tictools) [inline exercise_07/isTypeOf.js:21]: Avoid using `switch` when you can implement more simple logic using `if`.

FIX: needs work

**Comment by tictools** [comment]
@aalvar0 : you missed adding function signatures using `JSDocs` convention.

FIX: add function signatures

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by Manel0712 — CHANGES_REQUESTED** [review]
CI/CD
In github actions, run the lint:fix script, and the lint script should be run, not lint:fix

**Review by Manel0712 — COMMENTED** [review]
NPM
Everything is well done

**Review by Manel0712 — COMMENTED** [review]
Modules
Everything is well done

**Review by Manel0712 — CHANGES_REQUESTED** [review]
Function description missing

File: exercise_02/getTotalWordsFromString.js
Comment (Manel0712) [inline exercise_02/getTotalWordsFromString.js:1]: Function description missing

**Review by Manel0712 — CHANGES_REQUESTED** [review]

File: exercise_02/getTotalWordsFromString.js [hunk 3]
```diff
@@ -3,3 +3,21 @@
  * input: string (sentence)
  * output: number
  */
+
+/**
+ *
```
Comment (Manel0712) [inline exercise_02/getTotalWordsFromString.js:8]: Function description missing

**Review by Manel0712 — CHANGES_REQUESTED** [review]

File: exercise_02/getTotalWordsFromString.js [hunk 4]
```diff
@@ -3,3 +3,21 @@
  * input: string (sentence)
  * output: number
  */
+
+/**
+ * 
+ * @param {string} sentence 
+ * @returns {number}
+ */
```
Comment (Manel0712) [inline exercise_02/getTotalWordsFromString.js:11]: Function description missing

**Review by Manel0712 — CHANGES_REQUESTED** [review]

File: exercise_03/generateRandomNumberInRange.js [hunk 5]
```diff
@@ -3,3 +3,17 @@
  * input: number (limit)
  * output: number
  */
+
+/**
+ * 
+ * @param {number} limit 
+ * @returns {number}
+ */
```
Comment (Manel0712) [inline exercise_03/generateRandomNumberInRange.js:11]: Function description missing

File: exercise_04/isGreaterThan.js [hunk 6]
```diff
@@ -4,3 +4,14 @@
  * input: number (threshold)
  * output: boolean
  */
+/**
+ * 
+ * @param {number} value 
+ * @param {number} threshold 
+ * @returns {number}
+ */
```
Comment (Manel0712) [inline exercise_04/isGreaterThan.js:12]: Function description missing

File: exercise_05/isAdditionGreaterThanFifty.js [hunk 7]
```diff
@@ -4,3 +4,16 @@
  * input: number (summand2)
  * output: boolean
  */
+
+/**
+ * 
+ * @param {number} summand1 
+ * @param {number} summand2 
+ * @returns {boolean}
+ */
```
Comment (Manel0712) [inline exercise_05/isAdditionGreaterThanFifty.js:13]: Function description missing

File: exercise_06/checkCollectionHasElements.js [hunk 8]
```diff
@@ -3,3 +3,14 @@
  * input: array (collection)
  * output: boolean
  */
+
+/**
+ * 
+ * @param {Array} collection 
+ * @returns {boolean}
+ */
```
Comment (Manel0712) [inline exercise_06/checkCollectionHasElements.js:11]: Function description missing

File: exercise_07/isTypeOf.js [hunk 9]
```diff
@@ -6,3 +6,28 @@
  * input: boolean (type)
  * output:
  */
+
+/**
+ *
+ * @param {string|number|boolean|Array} value
+ * @param {boolean} type
+ * @returns
+ */
```
Comment (Manel0712) [inline exercise_07/isTypeOf.js:15]: Function description missing

**Review by Manel0712 — CHANGES_REQUESTED** [review]

**Review by Manel0712 — CHANGES_REQUESTED** [review]
Files are not formatted correctly

**Review by Manel0712 — APPROVED** [review]
Well Done!

File: package.json
Comment (tictools) [inline package.json:1]: FIX: Remove husky and lint-staged. Then merge

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by tictools — APPROVED** [review]

**Pull request merged**